### ISSUE TITLE
Add icons paths for icons used in validation/admin tools/gallery to util file

### ISSUE THREAD

**Issue by ThatOneGoat** [ProjectSidewalk/SidewalkWebpage#2442]
Right now there are some icon paths for icons used in the validation/admin tools/gallery pages that should probably be moved to a general util file.

**Comment by misaugstad** [comment]
@ThatOneGoat can you point to some of those so that this issue is more clear for someone in the future?

**Comment by ThatOneGoat** [comment]
Yup! Here is an example code block you can find in 'Card.js' from Gallery or 'Label.js" from Validation (also on the admin tools):
```
let iconImagePaths = {
        CurbRamp : '/assets/images/icons/AdminTool_CurbRamp.png',
        NoCurbRamp : '/assets/images/icons/AdminTool_NoCurbRamp.png',
        Obstacle : '/assets/images/icons/AdminTool_Obstacle.png',
        SurfaceProblem : '/assets/images/icons/AdminTool_SurfaceProblem.png',
        Other : '/assets/images/icons/AdminTool_Other.png',
        Occlusion : '/assets/images/icons/AdminTool_Other.png',
        NoSidewalk : '/assets/images/icons/AdminTool_NoSidewalk.png'
    };
```
We should probably move this object to a more generic util location so this block isn't just copy-pasted across the codebase.

**Comment by misaugstad** [comment]
This is essentially done. `util.misc.getIconImagePaths()` in `public/js/common/utilitiesSidewalk.js` is now the single source for label-type icon paths — it generates them from `VALID_LABEL_TYPES` rather than a hand-written map, and all 13 call sites across Explore, Validate, Gallery, admin, user-dashboard, shared-label, and label-detail go through it. CLAUDE.md documents it as the one way frontend code should name an icon.

One copy is left: `public/js/user-dashboard/src/MistakeCarousel.js:89` still builds `/assets/images/icons/AdminTool_${labelType}.png` — the exact legacy icon set this issue was filed about. It only runs on `/admin/user/:username`, which renders the pre-redesign dashboard, and #4964 will delete that page and its JS outright.

Closing in favor of #4964 rather than fixing a line that's about to be removed. The ten dead `public/images/icons/AdminTool_*.png` files go with it.
